props/websubmit/src/main.rs

```diff
@@ -273,8 +286,9 @@ fn run_dis_policy(ctx: Arc<Context>) -> Result<()> {
 fn main() -> Result<()> {
     let ws_dir = std::env::args()
         .nth(1)
-        .ok_or_else(|| anyhow!("expected format: cargo run <path> [policy]"))?;
-    let prop_name = std::env::args().nth(2);
+        .ok_or_else(|| anyhow!("expected format: cargo run <path> [edit-<property>-<articulation point>-<short edit type> | none] [policy]"))?;
+    let edit_name = std::env::args().nth(2);
+    let prop_name = std::env::args().nth(3);
```

Comment:
I think at this point it might be worth it to add some simple, derived `clap` arguments.

Reply:
done!